plugins/kubernetes/app/models/kubernetes/deploy_yaml.rb

```diff
@@ -1,8 +1,8 @@
 module Kubernetes
   class DeployYaml
     CUSTOM_UNIQUE_LABEL_KEY = 'rc_unique_identifier'.freeze
-    DAEMON_SET = 'DaemonSet'.freeze
-    DEPLOYMENT = 'Deployment'.freeze
+    SIDECAR_NAME = 'secret-sidecar'
+    SIDECAR_IMAGE = ENV['SECRET_SIDECAR_IMAGE']
```

Comment:
How about we add `presence` here, in case someone sets the ENV var to be an empty string?